src/thermobench.cpp

```diff
@@ -203,6 +204,35 @@ void write_arr_csv(FILE *fp, double *arr, int size)
     fprintf(fp, "\n");
 }
 
+void wait_cooldown(char* exec_dir){
+
+    FILE *fp = fopen(state.sensors[0].path, "r");
+    if (fp == NULL)
+        err(1, "Error while opening sensor file: %s\n", state.sensors[0].path);
+
+    char* cmd;
+    asprintf(&cmd,"%s/fan_ctrl.sh 1", exec_dir);
+    if(system(cmd) == -1)
+        err(1, "Error while executing shell command: %s\n", cmd);
+
+    while(1){
+        rewind(fp);
+        int temp = INT_MAX;
+        if(fscanf(fp, "%d", &temp) < 1) continue;
+        if(temp <= cooldown_temp) break;
+        printf("Cooling down to %d, CPU temperature: %d\n",cooldown_temp,temp);
```

Comment:
Print information messages to stderr. Later, we might want to use stdout for .csv. And is it really useful to print it every second? Maybe, you can use `"\rCooling..."` instead of `"Cooling...\n".